modules/appeals_api/config/schemas/v2/200995.json

```diff
@@ -127,10 +127,12 @@
                     "evidenceType":{
                       "type": "array",
                       "description": "'upload' indicates that the Veteran has supplied additional evidence as uploaded documents. 'retrieval' indicates that the Veteran has supplied contact information for facilities holding evidence relevant to the claim; the 'retrieveFrom' field will contain the facility's contact information.",
```

Comment:
I think we should include documentation about the new `none` value in this description, otherwise I don't think I could figure out that `none` was a valid value from the docs

Reply:
missed that thank you fixed

Reply:
Good catch thank you